src/Components/getList.js

```diff
@@ -1,22 +1,29 @@
 import React from 'react';
-import { FirestoreDocument } from 'react-firestore';
+import { FirestoreCollection } from 'react-firestore';
+import { getLocalToken } from '../lib/token.js';
 
 class GetList extends React.Component {
   render() {
     return (
-      <FirestoreDocument
-        path="items/tIA3kp24eSWYTNKxA3wl"
+      <FirestoreCollection
```

Comment:
This makes more sense now understanding a little more about firebase and structuring collections to users rather than documents.  Thank you for updating this

Reply:
Ya! @Tanzi11 really did a great job here.